packages/resolve-query/src/index.js

```diff
@@ -0,0 +1,7 @@
+export default ({ eventStore, projection }) => {
+    const eventsNames = Object.keys(projection.handlers);
+    let state = projection.initialState;
+    return () => eventStore.loadEventsByTypes(eventsNames, (event) => {
+        state = projection.handlers[event.__type](state, event);
```

Comment:
There should be check that `projection.handlers[event.__type]` is defined (generally not all possible events are handled in a projection).